walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +271,36 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+    @Test
+  public void nestedEventsNonNestedFollows() {
+    // Have an event for each person, but have one person's event fully contain another's event. Then,
+    // have a gap before a final event. We should see three options.
+    //
+    // Events  :      |----A----|   |-C-|
+    //                  |--B--|
+    // Day     : |---------------------------|
+    // Options : |-1-|          |-2-|    |-3-|
+
+    Collection<Event> events = Arrays.asList(
```

Comment:
Will you be adding more tests, perhaps in a separate PR?

Reply:
Yes! PR #24 has additional tests.

Reply:
sounds good